### PR TITLE
posix.mak: Enforce whitespace before opening version parenthesis

### PR DESCRIPTION
**Pull request by ibuclaw**
Get everyone using the same style.

### CODE REVIEW

**Comment by n8sh** [comment]
If this is being added to posix.max there should also be a line about it in https://github.com/dlang/dlang.org/blob/master/dstyle.dd. I recently checked it for guidance on `version` but couldn't find any.

File: etc/c/odbc/sqltypes.d [hunk 1]
```diff
@@ -175,11 +175,11 @@ alias GUID     = SQLGUID;
 alias BOOKMARK = uint;
 alias SQLWCHAR = ushort;
 
-version( UNICODE )
+version ( UNICODE )
```
Comment (wilzbach) [inline etc/c/odbc/sqltypes.d]: Could we use this opportunity to normalize these too?

**Comment by wilzbach** [comment]
I'm all in favor of being consistent in the style, but may I ask why you picked this one?

```
> ~/dlang/phobos on master ± ag "version\(" | wc -l
714

> ~/dlang/phobos on master ± ag "version \(" | wc -l
545
```

**Comment by ibuclaw** [comment]
> I'm all in favor of being consistent in the style, but may I ask why you picked this one?

You mean `version` style?  Because it is a heavily used feature that has at least three different styles, frequently intermixed in the same condition statement!

**Comment by wilzbach** [comment]
> You mean version style? 

No I meant why `version(` and not `version )` - see the output of the grep command.

**Comment by ibuclaw** [comment]
Ah, that would mean having its own regexp to catch that particular style.

I could use the same argument against the following also:
```
if ( cond )
switch ( cond )
while ( cond )
foreach ( var; array )
```

But to be really precise about this, we may need something smarter, as you'd also want to catch e.g:
```
if ( cond1 ||
     cond2 )
```

**Comment by wilzbach** [comment]
Yeah, you want this to be a D-Scanner check, e.g.

https://github.com/dlang-community/D-Scanner/pull/450
https://github.com/dlang-community/D-Scanner/pull/348

However, I never managed to push these into D-Scanner (there were some controversies on how D-Scanner can access the tokens), but we now have a special branch `phobos` there into which we can push anything (even things Brian objects to).

**Comment by n8sh** [comment]
(superseded by https://github.com/dlang/phobos/pull/6717)

**Pull request closed without merging**